package.json

```diff
@@ -4,64 +4,50 @@
   "version": "1.0.0",
   "license": "Apache-2.0",
   "scripts": {
-    "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
```

Comment:
The [Preact 10 docs](https://preactjs.com/guide/v10/getting-started) suggest the convention is to use `npm run dev`, so I can update the README to replace `npm start` with `npm run dev` (which worked fine when I tested it).

Reply:
Good catch, I went ahead and replaced it in the README myself.